scripts/install.js

```diff
@@ -48,7 +48,7 @@ function download(url, dest, cb) {
     return response.statusCode >= 200 && response.statusCode < 300;
   };
 
-  console.log('Start downloading binary at', url);
+  log.http('Start downloading binary at', url);
```

Comment:
I think this makes sense, but the messages need to be updated since the first parameter to the function in npmlog is the [`prefix`](https://www.npmjs.com/package/npmlog#loglevelprefix-message-) rather than just contating the series of objects
EX:
```js
log.http('install.js', 'Start downloading binary at ' + url)
```